scripts/copper/slab/analyze.py

```diff
@@ -12,7 +13,7 @@
 
 VACUUM_LAYER_PATH = PosixPath("data/copper/slab/vacuum_layer")
 FREE_LAYER_PATH = PosixPath("data/copper/slab/free_layer")
-
+SLAB_WIDTH_PATH = PosixPath("data/copper/na/width")
```

Comment:
should move this na specific stuff to the na folder. I would use the full name (sodium) rather than na everywhere too